app/assets/stylesheets/style.scss

```diff
@@ -18,22 +18,22 @@ body {
   background: #FAFAFA;
   // background: radial-gradient(ellipse at center, $color_steel_gray_approx 0, $color_hunter_green_approx 100%);
   color: $blue;
- a{
-  color:$blue;
-  font-family: $font_0;
- }
+  a{
+    color:$blue;
+    font-family: $font_0;
```

Comment:
Name of variable `font_0` should be written in all lowercase letters with hyphens instead of underscores